array-manipulation-2/src/take.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable no-unused-vars */
+
+function take(array, count) {
+  var newArray = [];
+  for (var i = 0; i < count; i++) {
+    if (array[i] !== undefined) {
+      newArray.push(array[i]);
+    }
+    continue;
```

Comment:
you don't need continue here, the loop will continue whether you tell it to or not.